lib/rubygems/errors.rb

```diff
@@ -20,6 +20,41 @@ class LoadError < ::LoadError
     attr_accessor :requirement
   end
 
+  class MissingSpecError < Gem::LoadError
+    def initialize name, requirement
```

Comment:
They use `()` around arguments in methods [definition](https://github.com/tenderlove/rubygems/blob/faster_miss/lib/rubygems/errors.rb#L146).